addons/l10n_latam_check/models/account_journal.py

```diff
@@ -0,0 +1,25 @@
+from odoo import models, fields, api, _
+from odoo.exceptions import UserError
+
+
+class AccountJournal(models.Model):
+    _inherit = 'account.journal'
+
+    l10n_latam_use_checkbooks = fields.Boolean(
```

Comment:
```suggestion
    l10n_latam_manual_checkbooks = fields.Boolean(
``` @vbe-odoo

Reply:
probably better to be l10n_latam_manual_checks instead of checkbooks?
@jco-odoo 
cc' @zaoral